frontend/ui/screensaver.lua

```diff
@@ -118,7 +118,7 @@ local function expandSpecial(message, fallback)
 
     local ReaderUI = require("apps/reader/readerui")
     local ui = ReaderUI:_getRunningInstance()
-    if ui then
+    if ui and ui.document then
```

Comment:
This.

Reply:
Thanks. So basically, something like:
```suggestion
    dbg.dassert(ui ~= nil)
    if ui and ui.document then
```

Reply:
How can UI be nil when the screensaver is entered? That doesn't seem to make sense.

Reply:
`UI` will be `nil` in the FM, for instance ;).